fuzz/fuzz_dump.cpp

```diff
@@ -42,6 +42,9 @@ static void print_json(std::ostream& os, simdjson::dom::element element) noexcep
   case simdjson::dom::element_type::UINT64:
     os << element.get_uint64().value_unsafe() << endl;
     break;
+  case simdjson::dom::element_type::BIGINT:
+    os << element.get_bigint().value_unsafe() << endl;
```

Comment:
This appears to be unsafe according to our CI.

Reply:
Is this because it's not '\0' terminated?

Reply:
I'm working on it too.

Reply:
I don't think your fix is correct. The issue here is where do you store the data in the DOM model. You cannot rely on the document being around.

Reply:
Yes, the commit below was my initial thought this morning, but right after that I realized the problem was related to the code inside `tape_writer::append_bigint(...)`, which you also noted down below.  I haven't had time this morning to investigate it further.  Will try to check this tonight.